src/Tribe/Assets.php

```diff
@@ -207,46 +207,52 @@ public function enqueue( $forcibly_enqueue = null ) {
 	 * If the file does not exist, returns false.
 	 *
 	 * @since 4.3
+	 * @since TBD Removed ability to pass a filepath as $url
 	 *
-	 * @param string $url   The path or URL to the un-minified file.
+	 * @param string $url The absolute URL to the un-minified file.
 	 *
-	 * @return string|false The path/url to minified version or false, if file not found.
+	 * @return string|false The url to the minified version or false, if file not found.
 	 */
 	public static function maybe_get_min_file( $url ) {
 		$urls = array();
-		// If need add the Min Files
-		if ( ! defined( 'SCRIPT_DEBUG' ) || SCRIPT_DEBUG === false ) {
-			if ( substr( $url, - 3, 3 ) === '.js' ) {
-				$urls[] = substr_replace( $url, '.min', - 3, 0 );
-			}
 
-			if ( substr( $url, - 4, 4 ) === '.css' ) {
-				$urls[] = substr_replace( $url, '.min', - 4, 0 );
-			}
+		if ( 0 === strpos( $url, WPMU_PLUGIN_URL ) ) {
+			// URL inside WPMU plugin dir.
+			$plugins_dir = wp_normalize_path( WPMU_PLUGIN_DIR );
+			$base_url   = WPMU_PLUGIN_URL;
+		} elseif ( 0 === strpos( $url, WP_PLUGIN_URL ) ) {
+			// URL inside WP plugin dir.
+			$plugins_dir = wp_normalize_path( WP_PLUGIN_DIR );
+			$base_url   = WP_PLUGIN_URL;
+		} else {
+			// Resource needs to be inside a plugins dir, if not we can't check if it exists.
```

Comment:
I know it is a super weird edge case, but some folks bake plugins inside of themes. I've had to do that when hosting a site on VIP.